inception-imls-stringmatch/src/main/java/de/tudarmstadt/ukp/inception/recommendation/imls/stringmatch/trie/Trie.java

```diff
@@ -36,7 +36,7 @@
 public class Trie<V>
 // implements Map<CharSequence, V>
 {
-    private int _size = 0;
+    private int s_size = 0;
```

Comment:
Why is the new name better than the old?

Reply:
FYI: I wrote this code a long time ago in the context of a different project. At that time, I used the convention that fields start with an underscore which would make it easy to distinguish fields from parameters and variables without having to prefix them with `this.`. We do not use this convention in INCEpTION though, but I didn't bother to update the file. According to the conventions we use these days, the underscore would simply be removed.